core/commands/configure_command.rb

```diff
@@ -66,11 +96,40 @@ def configure_rhel
     SUCCESS_RESULT
   end
 
+  def input_docker_credentials
+    if @configuration['docker'].nil?
```

Comment:
Please use Hash#dig method to fetch the contents of the configuration. This will allow to access the underlying key even if the parent key returns nil.

For example

```ruby
data = {'some' => { 'embedded' => 'value' } }
data.dig('some', 'embedded') # => 'value'
data.dig('none', 'none', 'none') # => nil
```

This way you won't need to duplicate the calls to the read_topic method.